blocks/product-gallery-block/index.story.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { ProductGalleryDisplay } from "./features/product-gallery/default";
+
+export default {
+	title: "Blocks/Product Gallery",
+	parameters: {
+		chromatic: { viewports: [320, 1200] },
+	},
```

Comment:
We should add the following here to ensure chromatic has the correct output

```
		cssVariables: {
			theme: "commerce",
		},
```

Reply:
will do, thanks. just noticed you did this before

Reply:
updated -- thanks